p2p/fuzz/src/lib.rs

```diff
@@ -326,7 +326,7 @@ impl NetworkScheme for Lookup {
             .topo
             .peers
             .iter()
-            .map(|p| (p.public_key.clone(), p.address))
+            .map(|p| (p.public_key.clone(), Address::from(p.address)))
```

Comment:
consistency: I think you use `.into()` most other places rather than `Address::from`